test/controllers/admin/users_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.